src/index.scss

```diff
@@ -1,15 +1,35 @@
-@import "~@edx/brand/paragon/fonts";
-@import "~@edx/brand/paragon/variables";
+// @import "~@edx/brand/paragon/fonts";
```

Comment:
I guess this is a WIP PR, but please be aware of the commented code.

Reply:
Thanks for looking. The changes in here weren't required in the end, so I'm closing the PR.